drivers/quota/projectquota.go

```diff
@@ -75,6 +78,25 @@ type Control struct {
 	quotas            map[string]uint32
 }
 
+// Attempt to generate a unigue projectid.  Multiple directories
+// per file system can have quota and they need a group of unigue
+// ids. This function attempts to allocate at least minPorjectIDS(10000)
+// unigue projectids, based on the inode of the basepath.
+func generateUnigueProjectID(path string) (uint32, error) {
+	const minProjectids = 10000
+	fileinfo, err := os.Stat(path)
+	if err != nil {
+		return 0, err
+	}
+	stat, ok := fileinfo.Sys().(*syscall.Stat_t)
+	if !ok {
+		return 0, fmt.Errorf("Not a syscall.Stat_t %s", path)
+
+	}
+	projectID := uint32(((stat.Ino * minProjectids) % math.MaxUint32))
+	return projectID, nil
+}
+
 // NewControl - initialize project quota support.
 // Test to make sure that quota can be set on a test dir and find
 // the first project id to be used for the next container create.
```

Comment:
This documentation needs updating.

Reply:
Updated and added information to the man page.